src/topology/local_homeomorph.lean

```diff
@@ -318,6 +318,9 @@ lemma of_set_target : (of_set s hs).target = s := rfl
 @[simp, mfld_simps] lemma of_set_coe : (of_set s hs : α → α) = id := rfl
 @[simp, mfld_simps] lemma of_set_symm : (of_set s hs).symm = of_set s hs := rfl
 
+lemma refl_eq_of_set : local_homeomorph.refl α = of_set univ is_open_univ :=
```

Comment:
I'd swap lhs with rhs and made it a `simp` lemma. @sgouezel  knows whether it should be `simp` or `simp, mfld_simps`.

Reply:
Done -- so far just `simp`.

Reply:
In these files, everything that is tagged `simp` should also be tagged `mfld_simps`, as a general rule.

Reply:
Done, thanks.